src/main/frontend/dicts.cljc

```diff
@@ -220,6 +220,7 @@
         :settings-page/disable-sentry "Send usage data and diagnostics to Logseq"
         :settings-page/disable-sentry-desc "Logseq will never collect your local graph database or sell your data." 
         :settings-page/preferred-outdenting "Logical outdenting"
+        :settings-page/show-full-blocks "Show full blocks"
```

Comment:
May finetune the text?
e.g. `Showing full content instead of first line for block reference`

Reply:
A minor addition to cnrpman comment;
What do you think of using the single form of blocks 
e.g.
```cljs
:settings-page/show-full-blocks "Show full content of a block refrence instead of first line"
```

Reply:
Changed in both english and spanish!

Reply:
Sorry but it's a bit too long 🤣 need to find a balance
Another proposal:
`Show all lines of block reference`
<img width="283" alt="image" src="https://user-images.githubusercontent.com/9862022/209830214-92dfe445-8202-4f90-a2b6-743847ae9d7b.png">

Reply:
Yeah, I found it a bit long too haha, pushing right now to correct it (force pushing so that we dont end up with 20 commits for this xD)

Reply:
Done, should be shorter now :)